storage/storage.c

```diff
@@ -1676,7 +1721,9 @@ static secbool storage_upgrade(void) {
         }
       }
     }
-  } else {
+  } else
+#endif
+  {
     // Copy all entries.
     uint32_t offset = 0;
     while (sectrue == norcow_get_next(&offset, &key, &val, &len)) {
```

Comment:
Some code blocks further down in this function can be go under `#if NORCOW_MIN_VERSION < ...`

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/f2cbe43d04de2fee44c84bbe1de62e13ad24c44d